### PR TITLE
CXX-762 - Audit bsoncxx::v0::types::value

### PR DESCRIPTION
**Pull request by tessavitabile**

### CODE REVIEW

File: src/bsoncxx/types/value.cpp [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// audit
```
Comment (amidvidy) [inline src/bsoncxx/types/value.cpp]: detritus?

Comment (samantharitter) [inline src/bsoncxx/types/value.cpp]: This was just so we could see the .cpp file in the review, to facilitate an audit of the code.

**Comment by acmorrow** [comment]
Sigh. Github pull request fail. You can't comment on lines you didn't change. We should probably move this review to reitveld.

**Comment by tessavitabile** [comment]
https://mongodbcr.appspot.com/42090001

**Pull request merged**